### PR TITLE
Adding addServers method to enable pool of servers for memcache

### PR DESCRIPTION
**Pull request by jaskaransingh156**
Adding addServers method to enable pool of servers for memcache

### CODE REVIEW

**Comment by jaskaransingh156** [comment]
It is related to the NFR mentioned in the following link:

https://github.com/phalcon/cphalcon/issues/1299

**Comment by sergeyklay** [comment]
For indents we use tabs in zep files instead of spaces

**Comment by jaskaransingh156** [comment]
Thanks sergeyklay for your feedback. I have corrected the indentation and committed the changes.

File: phalcon/cache/backend/memcache.zep [hunk 1]
```diff
@@ -117,6 +117,28 @@ class Memcache extends Backend implements BackendInterface
...
 
+        /**
+	 * Add servers to memcache pool
+	 *
+	 * @param  string host
+	 * @param  long port
+         * @param  boolean persistent	 
+         * @return boolean
+	 */
+	public function addServers(var host, var port, var persistent = false)
```
Comment (sergeyklay) [inline phalcon/cache/backend/memcache.zep]: I think should use such declaration:
```zephir
public function addServers(string! host, int port, boolean persistent = false) -> boolean
```

and remove `@param`s doc

See please PHP.net doc:

```php
bool Memcache::addServer ( string $host [, int $port = 11211 [, bool $persistent [, int $weight [, int $timeout [, int $retry_interval [, bool $status [, callable $failure_callback [, int $timeoutms ]]]]]]]] )
```

What with other params?

**Comment by sergeyklay** [comment]
Update please `CHANGELOG.md`

**Comment by jaskaransingh156** [comment]
Thanks @sergeyklay. I have changed declaration of addServers and removed @params doc and also updated CHANGELOG.md file. Regarding other arguments of addServer method, currently only 3 options (host, port and persistent) are being accepted in the constructor of Phalcon\Cache\Backend\Memcache. This host acts as the first server which is added to the memcache pool.  So, if i provide other arguments in addServers method then constructor should be also be changed accordingly in order to maintain consistency among first and all other servers. In that case, instead of using connect method of Memcache library, we should use addServer method to add even the first server. This has been explained in the following comment also:

http://php.net/manual/en/memcache.addserver.php#Hcom104484

Please confirm if constructor of Phalcon\Cache\Backend\Memcache is to be changed so that i will make the changes accordingly.

**Comment by sergeyklay** [comment]
Could you please rebase?

**Comment by jaskaransingh156** [comment]
@sergeyklay Done.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**